packages/tx-construction/src/types.ts

```diff
@@ -0,0 +1,87 @@
+import * as Crypto from '@cardano-sdk/crypto';
+import { AsyncKeyAgent, GroupedAddress, SignTransactionOptions, TransactionSigner } from '@cardano-sdk/key-management';
+import { Cardano } from '@cardano-sdk/core';
+import { InputSelector, SelectionSkeleton } from '@cardano-sdk/input-selection';
+import { Logger } from 'ts-log';
+
+import { MinimumCoinQuantityPerOutput } from './output-validation';
+
+export type InitializeTxResult = Cardano.TxBodyWithHash & { inputSelection: SelectionSkeleton };
+
+export interface Delegatee {
```

Comment:
This, `StakeKeyStatus` and `RewardAccount` would be better in `core`, as they're out of place in `tx-construction`